src/model_interface.jl

```diff
@@ -24,3 +25,34 @@ This function must be implemented for each calibration experiment.
 function observation_map(iteration)
     error("observation_map not implemented")
 end
+
+"""
+    postprocess_hook(ekp, g_ensemble, output_dir, iteration, prior)
+
+After each evaluation of the observation map and before updating the ensemble,
+`post_observation_map_hook` is evaluated.
+
+This function is optional to implement.
+
+For example, one may want to print information from the `eki` object or plot
+`g_ensemble`.
+"""
+function post_observation_map_hook(
+    ekp,
+    g_ensemble,
+    prior,
+    output_dir,
+    iteration,
+)
+    return nothing
```

Comment:
Could you make this log the mean constrained parameters and the error for the previous iteration by default?